### PR TITLE
Adding CGFloatEquals

### PR DESCRIPTION
**Pull request by marcelofabri**

### CODE REVIEW

File: CGFloatType/CGFloatType.h [hunk 1]
```diff
@@ -37,6 +37,9 @@ CGFloat CGFloatWithDouble(double value);
 /// Returns a CGFloat from a float
 CGFloat CGFloatWithFloat(float value);
 
+#pragma mark - CGFloat comparison
+
+extern BOOL CGFloatEquals(CGFloat x, CGFloat y);
```
Comment (kylef) [inline CGFloatType/CGFloatType.h]: Do we need the `extern` here, I though it doesn't have any effect in a framework header?

Comment (marcelofabri) [inline CGFloatType/CGFloatType.h]: I don't know, almost every function uses `extern` so I kept it. I can remove it if you think it's better

Comment (kylef) [inline CGFloatType/CGFloatType.h:42]: Hmm, your right. I didn't see those ones from this PR. Just stood out because the others shown in PR didn't have it.

**Comment by marcelofabri** [comment]
Thanks! :tada:

**Comment by kylef** [comment]
Thanks for the pull request. I notice you made a pull request before and you seem to be using this library for a while. Would you like to help maintain it and have commit access?

I don't see there needing to be many changes since this is a pretty small library.

**Comment by marcelofabri** [comment]
Sure, I'd love to help! Let's reduce the bus factor:

|  | kylef/CGFloatType |
| --- | --- |
| 🍴 | 8 forks. |
| 🔭 | Few watchers (2). |
| 🌟 | 36 stars. |
| 🗓 | Created almost 2 years ago; last push 7 minutes ago. |
| 🍻 | 10 PRs: 10 closed; 0 opened; 100.0% PRs are closed. |
| 🛠 | Deletions to additions ratio: 17.51% (-1454/8302). |
| 📦 | 1 releases; latest release "1.3.0": 7 months ago. |
| 🚌 | Bus factor: 100.0%. Most likely one core contributor. |

**Pull request merged**